src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -31,20 +31,20 @@
       <_NETCoreApp30RuntimePackVersion>3.0.3</_NETCoreApp30RuntimePackVersion>
       <_NETCoreApp30TargetingPackVersion>3.0.0</_NETCoreApp30TargetingPackVersion>
       
-      <_NETCoreApp31RuntimePackVersion>3.1.10</_NETCoreApp31RuntimePackVersion>
+      <_NETCoreApp31RuntimePackVersion>3.1.11</_NETCoreApp31RuntimePackVersion>
       <_NETCoreApp31TargetingPackVersion>3.1.0</_NETCoreApp31TargetingPackVersion>
       
       <_WindowsDesktop30RuntimePackVersion>3.0.3</_WindowsDesktop30RuntimePackVersion>
       <_WindowsDesktop30TargetingPackVersion>3.0.0</_WindowsDesktop30TargetingPackVersion>
       
-      <_WindowsDesktop31RuntimePackVersion>3.1.10</_WindowsDesktop31RuntimePackVersion>
+      <_WindowsDesktop31RuntimePackVersion>3.1.11</_WindowsDesktop31RuntimePackVersion>
       <_WindowsDesktop31TargetingPackVersion>3.1.0</_WindowsDesktop31TargetingPackVersion>
       
       <_AspNet30RuntimePackVersion>3.0.3</_AspNet30RuntimePackVersion>
       <_AspNet30TargetingPackVersion>3.0.1</_AspNet30TargetingPackVersion>
       
-      <_AspNet31RuntimePackVersion>3.1.10</_AspNet31RuntimePackVersion>
-      <_AspNet31TargetingPackVersion>3.1.10</_AspNet31TargetingPackVersion>
+      <_AspNet31RuntimePackVersion>3.1.11</_AspNet31RuntimePackVersion>
+      <_AspNet31TargetingPackVersion>3.1.11</_AspNet31TargetingPackVersion>
```

Comment:
@wtgodbe My branding script updated both the runtime and targeting packs for aspnet since they matched.  It doesn't look like we do this for windows desktop or netcore.  Since it looks like you last updated the targeting pack, can you confirm if the runtime and targeting pack versions should always match for aspnet?

Reply:
They should not always match, since we don't always ship the targeting pack. We've had to patch it a few times, most recently in 3.1.10, so that version should stay 3.1.10 until we have to patch it again (which, hopefully, will be never)